src/main/pages/setup-docker/docker-multi-user.adoc

```diff
@@ -8,56 +8,59 @@ folder: setup-docker
 ----
 
-`~/.che-multiuser` is any local path. This is where Che data and configuration will be stored.
+Here:
 
-`${EXTERNAL_IP}` should be a public IP accessible to all users who will access the Che instance. You may drop `CHE_HOST` env if you are running Che locally and will access it from within the same network. In this case, Che CLI will attempt to auto-detect your server IP. However, auto-detection may produce erroneous results, especially in case of a complex network setup. If you run Che as a cloud server, i.e. accessible for external users we recommend explicitly providing an external IP for `CHE_HOST`.
+* `~/.che-multiuser` is any local path. The Che data and configuration is stored at this location.
 
-You *MUST* access multi-user Che using hostname/IP specified in `CHE_HOST` or manually add redirect URIs and Web Origins to link:user-management.html#che-and-keycloak[Keycloak client].
+* `${EXTERNAL_IP}` should be a public IP accessible to all users who will access the Che instance. If you are running Che locally and will access it from within the same network, you may omit the `CHE_HOST` environment variable from the command. In this case, Che CLI will attempt to auto-detect your server IP. However, auto-detection may produce erroneous results, especially in case of a complex network setup. If you run Che as a cloud server that is accessible by external users, we recommend explicitly providing an external IP for the `CHE_HOST` variable.
 
-[id="whats-under-the-hood"]
-== What’s Under the Hood
+You must access Che multi-user using the hostname/IP specified in the `CHE_HOST` variable or manually add redirect URIs and Web Origins to the link:user-management.html#che-and-keycloak[Keycloak client].
 
-With `CHE_MULTIUSER=true` Che CLI is instructed to generate a special Docker Composefile that will be executed to produce config and run:
+[id="the-che-multiuser-variable"]
+== The CHE_MULTIUSER variable
+
+The `CHE_MULTIUSER` variable when set to `true`, instructs the Che CLI to generate a special `Docker Compose` file that will be executed to produce the configuration and run the following:
 
 * Keycloak container with pre-configured realm and clients
-* PostgreSQL container that will store Keycloak and Che data
+* PostgreSQL container that will store the Keycloak and Che data
 * Che server container with a special build of multi-user Che assembly
 
 [id="deployment-diagram"]
-== Deployment Diagram
+== Deployment diagram
 
-Multi-user Che on Docker does not differ much from a link:openshift-multi-user.html#deployment-diagram[multi-user deployment on OpenShift] in terms of architecture and communication between services. There are a few differences though:
+Che multi-user on Docker does not differ much from a link:openshift-multi-user.html#deployment-diagram[multi-user deployment on OpenShift] in terms of architecture and communication between services. However, following are some of the differences:
```

Comment:
* does not differ much -- not good; reformulate, please... _'The foll. is a list of diffs between...:'
* ditch 'however'